packages/datadog_tracking_http_client/lib/src/tracking_http_client.dart

```diff
@@ -164,6 +164,16 @@ class DatadogTrackingHttpClient implements HttpClient {
     return request;
   }
 
+  @override
+  set connectionFactory(
+          Future<ConnectionTask<Socket>> Function(
+                  Uri url, String? proxyHost, int? proxyPort)?
+              f) =>
```

Comment:
should this be called just `f`?

Reply:
Yes, it's a straight override of the HttpClient function which has the terrible name.